src/main/scala/lib/foldM.scala

```diff
@@ -0,0 +1,13 @@
+package lib
+
+import cats.Monad
+
+import scala.language.higherKinds
+
+object foldM {
```

Comment:
sadly no... there is an open PR for it - https://github.com/non/cats/pull/356 but its on hold